src/app/inventory/d2-stores.ts

```diff
@@ -359,10 +359,7 @@ function processCharacter(
   const store = makeCharacter(defs, character, lastPlayedDate);
 
   // We work around the weird account-wide buckets by assigning them to the current character
-  const items = characterInventory.slice();
-  for (const k in characterEquipment) {
-    items.push(characterEquipment[k]);
-  }
+  const items = characterInventory.concat(characterEquipment.flat());
```

Comment:
Does this `flat()` do anything?

Reply:
It should? Each character inventory is a list of items.

Reply:
i'm with robo on this, either the types are wrong or this flat does nothing